CODEOWNERS

```diff
@@ -1,38 +1,26 @@
-/pkg/bootstrap/                                                  @mandarjog @costinm
-/pkg/mcp/                                                        @ozevren @ayj @nmittler
-/prow/                                                           @geeknoid @howardjohn @duderino @linsun
-/release/                                                        @duderino @linsun
-/samples/                                                        @frankbu @kyessenov @vadimeisenbergibm @venilnoronha
-/security/                                                       @diemtvu @incfly @myidpt @quanjielin @JimmyCYJ
-/tests/e2e/local/                                                @costinm @nmittler @gargnupur @kimikowang @JimmyCYJ @m1093782566
-/tests/e2e/tests/pilot/                                          @costinm @nmittler @andraxylia @howardjohn @GregHanson @hzxuzhonghu @rshriram @vadimeisenbergibm @zackbutcher
-/tests/integration/conformance/                                  @costinm @nmittler @ozevren
-/tests/integration/framework/                                    @costinm @nmittler @ozevren
-/tests/integration/galley/                                       @costinm @nmittler @ozevren @geeknoid
-/tests/integration/mixer/                                        @costinm @nmittler @ozevren @bianpengyuan @douglas-reid @gargnupur @geeknoid @kyessenov @mandarjog
-/tests/integration/pilot/                                        @costinm @nmittler @ozevren @andraxylia @costinm @howardjohn @GregHanson @hzxuzhonghu @nmittler @rshriram
-/tests/integration/qualification/                                @costinm @nmittler @ozevren
-/tests/integration/security/                                     @costinm @nmittler @ozevren @diemtvu @incfly @myidpt @quanjielin @wattli @yangminzhu @JimmyCYJ
-/tests/integration/security/rbac/                                @costinm @nmittler @ozevren @diemtvu @incfly @myidpt @quanjielin @wattli @yangminzhu @JimmyCYJ @yangminzhu @liminw
-/tests/istio.io/                                                 @rlenglet @brian-avery @JimmyCYJ @yangminzhu
-/tools/                                                          @sdake @mandarjog @nmittler
+*                                                                @geeknoid @linsun @rshriram
+/bin/                                                            @istio/wg-test-and-release-maintainers
+/galley/                                                         @istio/wg-config-maintainers
+/install/                                                        @istio/wg-environments-maintainers
+/istioctl/                                                       @istio/wg-user-experience-maintainers
+/mixer/                                                          @istio/wg-policy-and-telemetry-maintainers
+/pilot/                                                          @istio/wg-networking-maintainers
+/pilot/pkg/security/                                             @istio/wg-security-maintainers
```

Comment:
My thinking was a networking owner should be able to say "no" to any PR in this folder since it can impact all of pilot. But adding networking doesn't actually do that, as it would be (`networking || security`). I don't think there is a way to say  `networking && security`, and I think that would be a bit excessive anyways -- pilot owners can still review changes regardless.

So I think it makes sense how it is, and same for other ones like mixer, etc

Reply:
You can always “request_changes” and block the PR.
I hope that we make that possible.

Reply:
Yes that is definitely still possible which is why I think it's ok without networking owners now